connector.go

```diff
@@ -54,7 +54,7 @@ func (c *connector) Deploy(ctx context.Context, image string) (deployer.Plugin,
 
 func (c *connector) startContainer(ctx context.Context, cnt *connectorContainer) error {
 	c.logger.Debugf("Starting container %s...", cnt.id)
-	if err := c.cli.ContainerStart(ctx, cnt.id, types.ContainerStartOptions{}); err != nil {
+	if err := c.cli.ContainerStart(ctx, cnt.id, container.StartOptions{}); err != nil {
```

Comment:
So docker v25 has incompatible changes from v24, but the changes are restricted to the names of these three referenced types? Interesting ... I hope the CI tests this well!

Reply:
me too